modules/http/http_filters.c

```diff
@@ -1636,25 +1713,22 @@ AP_DECLARE(int) ap_discard_request_body(request_rec *r)
 
 AP_DECLARE(int) ap_setup_client_block(request_rec *r, int read_policy)
 {
-    const char *tenc = apr_table_get(r->headers_in, "Transfer-Encoding");
```

Comment:
This is the first place were we eliminate checks on `Transfer-Encoding`, because HTTP version > 1.1 do no longer use it.